module/VuFind/tests/unit-tests/src/VuFindTest/Form/FormTest.php

```diff
@@ -367,22 +367,25 @@ public function testSenderFieldMergingWithSettings()
                     'name' => 'name',
                     'group' => '__sender__',
                     'label' => 'Sender Name',
-                    'settings' => ['size' => 100],
+                    'settings' => [
+                        'maxlength' => 50, // from Form.php defaults
+                        'size' => 100, // from feedbackforms/test.yaml
```

Comment:
I know this is just a test, but why have a 100 characters wide field if you only can input 50? Can we revise to like maxlength=100 and size=50?

Reply:
This is all a result of a misunderstanding on my end on how `size` works. Would you agree that size and maxlength can be eliminated in favor of CSS?

Reply:
At least when feedback is stored in the database we should enforce maximum length somehow (unlike we now do), so setting maxlength to what the database supports is one sensible option. But that's not necessarily the scope of this PR, so I'd be fine with leaving both out here.